src/resolvers.ts

```diff
@@ -512,3 +530,111 @@ export async function resolveThirdPartyCollectionOnlyCollection(
 
   return result
 }
+
+export async function resolveLinkedWearableProviderName(
+  uri: URL,
+  groups: Record<'network' | 'linkedWearableProvider', string>
+): Promise<BlockchainCollectionLinkedWearablesName | undefined> {
+  let result: BlockchainCollectionLinkedWearablesName | undefined = undefined
+  if (!isValidNetwork(groups.network)) return
+
+  const contract = await getContract(groups.network, 'TPR')
+
+  if (contract) {
+    result = {
+      namespace: 'decentraland',
+      uri,
+      blockchain: 'ethereum',
+      type: 'blockchain-collection-linked-wearables-name',
+      network: groups.network === 'ethereum' ? 'mainnet' : groups.network.toLowerCase(),
+      thirdPartyName: groups.linkedWearableProvider,
+      contractAddress: contract
+    }
+  }
+
+  return result
+}
+
+export async function resolveLinkedWearableCollection(
+  uri: URL,
+  groups: Record<'network' | 'linkedWearableProvider' | 'contractAddressChain' | 'collectionId', string>
+): Promise<BlockchainCollectionLinkedWearablesCollection | undefined> {
+  let result: BlockchainCollectionLinkedWearablesCollection | undefined = undefined
+  if (!isValidNetwork(groups.network)) return
+
+  const contract = await getContract(groups.network, 'TPR')
+
+  if (contract) {
+    result = {
+      namespace: 'decentraland',
+      uri,
+      blockchain: 'ethereum',
+      type: 'blockchain-collection-linked-wearables-collection',
+      contractAddress: contract,
+      network: groups.network === 'ethereum' ? 'mainnet' : groups.network.toLowerCase(),
+      thirdPartyName: groups.linkedWearableProvider,
```

Comment:
Should we standardise how we are calling to this property?
```suggestion
      linkWearableProvider: groups.linkedWearableProvider,
```